public/pages/ConfigureModel/utils/helpers.ts

```diff
@@ -425,3 +438,138 @@ export const getCustomValueStrArray = (imputationMethodStr : string, detector: D
   }
   return []
 }
+
+export const getSuppressionRulesArray = (detector: Detector): string[] => {
+  if (!detector.rules || detector.rules.length === 0) {
+    return []; // Return an empty array if there are no rules
+  }
+
+  return detector.rules.flatMap((rule) => {
+    // Convert each condition to a readable string
+    return rule.conditions.map((condition) => {
+      const featureName = condition.featureName;
+      const thresholdType = condition.thresholdType;
+      let value = condition.value;
+      const isPercentage = thresholdType === ThresholdType.ACTUAL_OVER_EXPECTED_RATIO || thresholdType === ThresholdType.EXPECTED_OVER_ACTUAL_RATIO;
+
+      // If it is a percentage, multiply by 100
+      if (isPercentage) {
+        value *= 100;
```

Comment:
nit: could value be undefined or null here? Should we handle it here? something like `value = (value ?? 0) * 100;`

Reply:
I made sure the value is a positive number before saving it in detector: https://github.com/opensearch-project/anomaly-detection-dashboards-plugin/blob/1708b0b16684f28079cc36fa964b9e405ef0dddc/public/pages/ConfigureModel/utils/helpers.ts#L506-L518

On the backend, the default value is 0. If relative value is there, it has to be a double. So it cannot be undefined or null.